Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -249,7 +249,22 @@ OSD locations defined in the CRUSH map and topology labels on nodes.
 Refer to the [krbd-options](https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options)
 for more details.
 
-Execute the following steps:
+Execute the following step to enable read affinity for a specific ceph cluster:
+
+* Patch the ceph cluster CR to enable read affinity:
+
+```console
+kubectl patch cephclusters.ceph.rook.io <cluster-name> -n rook-ceph -p '{"spec":{"csi":{"readAffinity":{"enabled": true}}}}'
+```
+
+```yaml
+  csi:
+    readAffinity:
+      enabled: true
+```
+
+
+Execute the following steps to enable read affinity for all ceph clusters:
```

Comment:
How about we remove this operator config from the docs? Even if we still have it as a fallback/default setting, we want people to start using the new setting, and no need for anyone to use it, right?

Reply:
> How about we remove this operator config from the docs? Even if we still have it as a fallback/default setting, we want people to start using the new setting, and no need for anyone to use it, right?

Makes sense, Its removed now.